app/models/category.rb

```diff
@@ -1,2 +1,5 @@
 class Category < ApplicationRecord
+
+  has_many :posts
+  
```

Comment:
Please write tests for all models follow example for this model below:
- install gem that I commented on below
- create a new directory with a new file called:
  ``` /spec/models/post_spec.rb ```
- add the following code:
  ```
require "rails_helper"

RSpec.describe Post, type: :model do
  describe 'Assoociations' do
    it { should belong_to(:category) }
    it { should have_many(:comments) }
  end

  describe 'Database data types' do
    it { should have_db_column(:title).of_type(:string) }
    it { should have_db_column(:author).of_type(:string) }
    it { should have_db_column(:body).of_type(:text) }
    it { should have_db_column(:tags).of_type(:string) }
    it { should have_db_column(:user_id).of_type(:integer) }
    it { should have_db_column(:category_id).of_type(:integer) }
  end
end
```
- Do this same example for all models
- Give a 10 minute "lunch and learn" to everyone about what your doing here